homeassistant/components/spotify/media_player.py

```diff
@@ -408,24 +423,54 @@ async def async_browse_media(self, media_content_type=None, media_content_id=Non
         return response
 
 
-def build_item_response(spotify, payload):
+def build_item_response(spotify, user, payload):
     """Create response payload for the provided media query."""
     media_content_type = payload.get("media_content_type")
```

Comment:
```suggestion
    media_content_type = payload.get("media_content_type")
    media_content_id = payload.get("media_content_id")
```
And replace the other `payload.get("media_content_id")` thereafter.